src/SFA.DAS.EmployerFinance.Api.Client/Configuration/ConfigurationKeys.cs

```diff
@@ -0,0 +1,8 @@
+
+namespace SFA.DAS.EmployerFinance.Api.Client.Configuration
+{
+    public static class ConfigurationKeys
+    {
+        public const string ApiClientBase = "SFA.DAS.EmployerFinance.Api.ClientV2";
```

Comment:
What do you think to renaming this key to just `ApiClient`?

The `SFA.DAS.EmployerFinance.Api.ClientV2` config will also need adding to the `das-employer-config` repo, it might be worth dropping it on @lazcool's PR https://github.com/SkillsFundingAgency/das-employer-config/pull/187.

Reply:
ok cool i'll make those changes

Reply:
done. I've also added the config to the PR mentioned